### DISCUSSION TITLE
是否有plan去引进字节跳动的coze[Feature]

### DISCUSSION THREAD

**Discussion by Become-ILLUSORY** [ChatGPTNextWeb/NextChat#3955] [Ideas]
> 为了提高交流效率，我们设立了官方 QQ 群和 QQ 频道，如果你在使用或者搭建过程中遇到了任何问题，请先第一时间加群或者频道咨询解决，除非是可以稳定复现的 Bug 或者较为有创意的功能建议，否则请不要随意往 Issue 区发送低质无意义帖子。

> [点击加入官方群聊](https://github.com/Yidadaa/ChatGPT-Next-Web/discussions/1724)

**这个功能与现有的问题有关吗？**
如果有关，请在此列出链接或者描述问题。

**你想要什么功能或者有什么建议？**
尽管告诉我们。

**有没有可以参考的同类竞品？**
可以给出参考产品的链接或者截图。

**其他信息**
可以说说你的其他考虑。

[a comment by an automated account is not shown]

**Comment by sheng-di** [comment]
coze 真是太厉害了，支持 workflow、支持 JS 和 Python 节点，真的是不错。
